compaction/compaction_manager.cc

```diff
@@ -924,8 +924,7 @@ compaction_manager::compaction_manager(config cfg, abort_source& as, tasks::task
     , _update_compaction_static_shares_action([this] { return update_static_shares(static_shares()); })
```

Comment:
>  raft: refactor and simplify the repair map 

The prefix `raft: ` looks wrong, I would say this is more of a `compaction: ` or `tombstone_gc: ` thing.

Reply:
changed to `tombstone_gc:`